src/main/java/com/shopify/store/FruitProduct.java

```diff
@@ -0,0 +1,60 @@
+package com.shopify.store;
+
+
+/**
+ * FruitProduct Products
+ */
+public class FruitProduct implements Product{
+    private ProductType productType;
+    private String fruitName;
+    private long price;
+    private int quantity;
+
+    /**
+     *
+     * @param fruitName takes in the fruitName and in the future would implement unique SKU
+     * @param quantity how many of the fruit product
+     */
+    public FruitProduct (String fruitName, int quantity) {
```

Comment:
Why isn't price a param here?

Reply:
Hmmm Ahh, i think I can create a better structure here yes. I was concerned that it would have to take in [<product>,<quantity>] only.